packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -448,7 +448,6 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
      */
     function redeem(FixedPoint.Unsigned memory numTokens)
         public
-        onlyPreExpiration()
```

Comment:
General question: would it make sense to bound this to only be before the DVM has resolve? You can still `redeem` if the contract has been `expire`d but if the contract has settled & the DVM has resolved then force all users to go via `settleExpired`? This still gives users the ability to redeem after expiry but ensures that once the contract has a DVM price everyone is exiting via the same price & method.

I'm not sure if there is really and advantage to doing this; or put another way if there are any issues with leaving `redeem` accessible at the same time that `settleExpired` is accessible but just wanted to discuss it and the implications.

Reply:
It's a really good question/point. 

As far as I'm aware, there's no problem with allowing it to be called after settlement. I worked through the logic and added a test for this case, and I didn't notice any issues with having them work concurrently, but that's not to say it isn't a problem or could cause usability issues.

The downside to limiting it the way you suggest is that I think we'd need to add in price retrieval logic to the redeem method to determine if a price is available yet, which could add additional complexity. I think it would require something like this: https://github.com/UMAprotocol/protocol/blob/e02517dd2fd3ce7ae55e554c6002f9724fb605c9/packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol#L505-L511 (but I could be wrong!).

Thoughts?

Reply:
Well does it need to have this logic within this function? would it not be sufficient to simply block this function if we are after DVM has resolved and then force the use to use `settleExpired`?

Reply:
The second case within a modifier is pretty much what I was thinking.

Reply:
My slight preference would be to avoid the additional complexity of this restriction unless we think that allowing both to be active simultaneously would break something. But I'm open to adding the restriction if y'all feel strongly.